components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -102,8 +102,15 @@ - (void)loadView {
 - (void)viewWillAppear:(BOOL)animated {
   [super viewWillAppear:animated];
 
-  CGPoint point = [_highlightedView.superview convertPoint:_highlightedView.center
-                                                    toView:_featureHighlightView];
+  CGPoint point;
+  if ([self deviceUsesCoordinateSpaces]) {
+    // We have to use coordinate spaces on iOS8 devices
+    point = [_highlightedView.superview convertPoint:_highlightedView.center
+                                   toCoordinateSpace:_featureHighlightView];
+  } else {
```

Comment:
Won't all iOS version iOS 8+ will return YES for deviceUsersCoordinateSpaces?
If that is the case, this else clause will never be called.